src/UI/walletConnect/WalletConnectLoginContainer/PairingList.tsx

```diff
@@ -1,98 +1,94 @@
 import React, { useState, useEffect } from 'react';
 import { PairingTypes } from '@multiversx/sdk-wallet-connect-provider';
-import { faCircleNotch } from '@fortawesome/free-solid-svg-icons';
+import { faCircleNotch, faTimes } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import classNames from 'classnames';
-import globalStyles from 'assets/sass/main.scss';
 
-import { WithClassnameType } from '../../types';
+import type { WithClassnameType } from '../../types';
+
+import styles from './walletConnectLoginContainerStyles.scss';
+
+export interface WalletConnectPairingListClassesType {
+  walletConnectPairingListLeadClassName?: string;
+  walletConnectPairingListWrapperClassName?: string;
```

Comment:
here, and above, I would remove walletConnectPairingList from namespacing inside props because you already have the key offering the namespace

Reply:
Changed. 👍